crud/borders.lua

```diff
@@ -0,0 +1,180 @@
+local checks = require('checks')
+local errors = require('errors')
+local vshard = require('vshard')
+
+local dev_checks = require('crud.common.dev_checks')
+local call = require('crud.common.call')
+local utils = require('crud.common.utils')
+local schema = require('crud.common.schema')
+local Keydef = require('crud.compare.keydef')
+
+local BorderError = errors.new_class('Border',  {capture_stack = false})
+
+local borders = {}
+
+local STAT_FUNC_NAME = '_crud.get_border_on_storage'
+
+
+local function get_border_on_storage(border_name, space_name, index_id, field_names)
+    dev_checks('string', 'string', 'number', '?table')
```

Comment:
index_id could be string or number

Reply:
Here - no. See `get_border` function on router.